runtime/compiler/trj9/p/codegen/PPCJNILinkage.cpp

```diff
@@ -168,16 +166,10 @@ TR::Register *TR::PPCJNILinkage::buildDirectDispatch(TR::Node *callNode)
       wrapRefs = false; //unused for this code path
       passReceiver = true;
       passThread = false;
-      jniAccelerated = false; //unused for this code path
       targetAddress = (uintptrj_t)callSymbol->getMethodAddress();
       }
 
-   if (!isGPUHelper && (TR::Options::getJniAccelerator() != NULL))
-      {
-      jniAccelerated = TR::SimpleRegex::match(TR::Options::getJniAccelerator(), resolvedMethod->signature(comp()->trMemory()));
-      }
-
-   if (!isGPUHelper && (callSymbol->isPureFunction() || resolvedMethodSymbol->canDirectNativeCall() || jniAccelerated || specialCaseJNI))
+   if (!isGPUHelper && (callSymbol->isPureFunction() || resolvedMethodSymbol->canDirectNativeCall() || specialCaseJNI))
```

Comment:
@zl-wang there are 3 checks for `wrapRefs`, `passReceiver`, `passThread`. Are these not meant to be controlled by FastJNI?

Reply:
@fjeremic  VM-side recognized a set of FastJNI methods.  One of the checks reflects that control. But there are other cases VM doesn't know, e.g., Math.log() etc.   jniAcc is one of the cases as well.